tests/regression/tests/REQUEST-931-APPLICATION-ATTACK-RFI/931130.yaml

```diff
@@ -181,3 +181,51 @@ tests:
             version: HTTP/1.1
           output:
             no_log_contains: id "931130"
+  - test_title: 931130-12
+    desc: Bypass 1
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP ModSecurity Core Rule Set"
+              Host: example.com
```

Comment:
Do these tests require `example.com` as `Host` header?

Reply:
I think yes, or localhost or something random. the chained rule checks if the domain captured in group 1 ends with the request host header